### PR TITLE
ami : Added tdnn_lstm recipe with fast-lstmp layer. Added tdnn_lstm r…

### PR DESCRIPTION
**Pull request by vijayaditya**
…ecipe with -1 delay at lowest lstm layer

### CODE REVIEW

File: egs/ami/s5b/local/chain/tuning/run_tdnn_lstm_1j.sh [hunk 1]
```diff
@@ -0,0 +1,301 @@
+#!/bin/bash
+
+# 1j is same as 1i but with changes related to fast-lstmp layer
+# changed num-chunk-per-minibatch to be variable
+# added extra_left_context_initial=0
+# and extra_right_context_final=0
+# These changes are similar to those between swbd's run_tdnn_lstm_1{c,d}.sh
+# recipes
+
+#System            tdnn_lstm1i_sp_bi_ihmali_ld5 tdnn_lstm1j_sp_bi_ihmali_ld5
```
Comment (danpovey) [inline egs/ami/s5b/local/chain/tuning/run_tdnn_lstm_1j.sh]: I think we should probably mention for these AMI results whether it was for ihm, etc.

Comment (danpovey) [inline egs/ami/s5b/local/chain/tuning/run_tdnn_lstm_1j.sh]: and any chance of including the output of chain_dir_info.pl, for future reference?

File: egs/swbd/s5c/local/chain/tuning/run_tdnn_lstm_1j.sh [hunk 2]
```diff
@@ -0,0 +1,311 @@
+#!/bin/bash
+
+# same as 1e but with delay of -1
+# System                tdnn_lstm_1e_sp tdnn_lstm_1j_sp
+# WER on train_dev(tg)  12.74           12.95
```
Comment (vijayaditya) [inline egs/swbd/s5c/local/chain/tuning/run_tdnn_lstm_1j.sh]: @danpovey similar results even on SWBD

**Comment by danpovey** [comment]
Thanks!  I'll merge.

**Pull request merged**